066-OpenAIFundamentals/Student/Resources/Notebooks/CH-04-A-RAG_for_structured_data.ipynb

```diff
@@ -0,0 +1,1124 @@
+{
```

Comment:
Might be better to stay away from potentially sensitive news topics for this challenge (i.e., politics).

Reply:
Feedback noted. The authors will consider a different data source in vNext of this hack.